.github/ISSUE_TEMPLATE/release.md

```diff
@@ -37,6 +37,9 @@ production readiness.
 
 ### Create the release candidate
 
+⚠️ Make sure to run these commands on a direct clone of the MaterializeInc/Materialize repo.
+These commands will not run if you run them from your fork!
```

Comment:
Hmm, they work just fine for me on my fork. Regardless, we should probably fix this rather than documenting it?

Reply:
The issue with the instructions is the `git push origin` that assumes `origin` is `MaterializeInc/Materialize`, not a fork.

Reply:
What do your configuration look like? Can you post the results of `git remote -vv`?

Reply:
Oh I see, yeah, I still use "origin" as the name of the upstream. The `mkrelease` command does some magic to determine the correct name of the remote (the first remote that matches `MaterializeInc/materialize`). Maybe we can rephrase the instructions accordingly?

I'm actually not sure why `mkrelease` doesn't just push the tag, considering there isn't any sort of verification you have to do between creating the tag and pushing it. @quodlibetor do you have thoughts?

Reply:
(Anyway, I think this is fine for now.)

Reply:
Okay, I'll push now and we'll ask @quodlibetor to check the `mkrelease` script.

Reply:
oh it literally was just that I had already spent a day on it and it seemed easier to get reviewed if it had no risk of breaking the remote repo. I'll add a prompt-to-push.

Reply:
#4337